### PR TITLE
added AI readme 

### LINKED ISSUE #814: Title: I wish to update AI README file
### Explain what you want to add/change in documentation.

As of now readme file is empty and I want to add content according to the format needed.

### PR DESCRIPTION
**Pull request by muskan467**
### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md)?

Yes, I have read the Contributing guidelines.

### Description

I have made the AI readme.

Fixes #814 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#814

### CODE REVIEW

**Comment by muskan467** [comment]
Kindly merge it @Kushal997-das

File: Artificial Intelligence/README.md [hunk 1]
```diff
@@ -1 +1,49 @@
...
+<p align="center"><img src="https://www.srimax.com/wp-content/uploads/2020/01/Importance-of-Artificial-Intelligence.jpeg" alt="Artificial Intelligence Image" width=800px />
+<div id="top"></div>
+  
+<h2 align="center">🚦 Welcome Developers to Artificial Intelligence Domain of Project Guidance</p></h2>
+
+> <h3>🏰 In this directory, you will find awesome Artificial Intelligence projects of different levels. Passionate developers develop the projects to help beginners learn to program and practice effectively by building great projects.</h3>
+
+</br>
+
+<p align="right"><a href="#Bottom"><img src="https://img.shields.io/badge/-Bottom-red?style=for-the-badge" /></a></p>
```
Comment (Kushal997-das) [inline Artificial Intelligence/README.md:12]: @muskan467 Reduce the width and height of the image . Make it a small image .

**Review by Kushal997-das — CHANGES_REQUESTED** [review]
@muskan467 Do the changes .

**Review by Kushal997-das — APPROVED** [review]
@muskan467 Great work

**Pull request merged**